packages/block-library/src/navigation-link/style.scss

```diff
@@ -29,6 +29,9 @@
 // Styles for submenu flyout
 .has-child {
 	$navigation-vertical-padding: $grid-unit-10 * 0.75;
+	> a {
```

Comment:
If you target `.wp-block-navigation-link__content` instead of `a` in this file, it will apply to both editor and front end, so you won't need the change in `editor.scss` 🙂

Reply:
Thank you very much! I will work on it 😄 !